include/bluetooth/mesh/cfg_cli.h

```diff
@@ -211,6 +211,21 @@ int bt_mesh_cfg_relay_set(u16_t net_idx, u16_t addr, u8_t new_relay,
 int bt_mesh_cfg_net_key_add(u16_t net_idx, u16_t addr, u16_t key_net_idx,
 			    const u8_t net_key[16], u8_t *status);
 
+/** @brief Get a list of the target node's network key indexes.
+ *
+ *  @param net_idx Network index to encrypt with.
+ *  @param addr    Target node address.
+ *  @param keys    Net key index list response parameter. Will be filled with
+ *                 all the returned network key indexes it can fill.
+ *  @param key_cnt Net key index list length. Should be set to the
+ *                 capacity of the @c keys list when calling. Will return the
```

Comment:
@c what means?

Reply:
It'll make the next word render as code in the documentation